### PR TITLE
Feature/xivy 3565 bpm test rt properties

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

**Comment by ivy-rew** [comment]
cleans up the demos: however, we have to wait until other quality issues in the project-build-plugin have been addressed (which currently let the builds fail as soon as I switch to project-build-plugin:8.0.4-SNAP)

**Comment by ivy-rew** [comment]
still has a deployment issue: but I think we will overcome that with the next task :)

File: build.maven/iar/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<antrun.version>1.7</antrun.version>
-		<project.build.plugin.version>8.0.3</project.build.plugin.version>
+		<project.build.plugin.version>8.0.4-SNAPSHOT</project.build.plugin.version>
```
Comment (weissreto) [inline build.maven/iar/pom.xml]: Can you now use SNAPSHOT version here? 
Do you update later to 9.1.0-SNAPSHOT?

Comment (ivy-rew) [reply build.maven/iar/pom.xml]: parent poms are resolvable (so snapshots repos are public visible)
https://repo.axonivy.com/artifactory/libs/ch/ivyteam/ivy/project/build.maven.iar/

and as far as I know we no longer manipulate the final pom 's so it should work.

Comment (ivy-rew) [reply build.maven/iar/pom.xml]: and yes the final solution is to use 9.1.0-snap

**Comment by weissreto** [comment]
Feel free to merge

**Pull request merged**